src/v2/Apps/FairOrginizer/Routes/FairOrganizerDedicatedArticles.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Box,
+  Column,
+  Flex,
+  GridColumns,
+  Spacer,
+  Text,
+  Title,
+} from "@artsy/palette"
+import React, { useEffect, useState } from "react"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
+import { PaginationFragmentContainer as Pagination } from "v2/Components/Pagination"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { DedicatedArticlesBreadcrumbsFragmentContainer as DedicatedArticlesBreadcrumbs } from "../Components/DedicatedArticlesBreadcrumbs"
+import { FairOrganizerDedicatedArticlesQuery } from "./FairOrganizerDedicatedArticlesQuery"
+import { FairOrganizerDedicatedArticles_fairOrganizer } from "v2/__generated__/FairOrganizerDedicatedArticles_fairOrganizer.graphql"
+import createLogger from "v2/Utils/logger"
+import { updateUrl } from "v2/Components/ArtworkFilter/Utils/urlBuilder"
+import { LoadingArea } from "v2/Components/LoadingArea"
+import { FairOrganizerArticle } from "../Components/FairOrganizerArticle"
+
+const PAGE_SIZE = 16
+
+interface FairOrganizerDedicatedArticlesProps {
+  fairOrganizer: FairOrganizerDedicatedArticles_fairOrganizer
+  relay: RelayRefetchProp
+}
+
+export const FairOrganizerDedicatedArticles: React.FC<FairOrganizerDedicatedArticlesProps> = ({
+  fairOrganizer,
+  relay,
+}) => {
+  const { name, slug, articlesConnection } = fairOrganizer
+  const {
+    pageInfo: { hasNextPage },
+    pageCursors,
+  } = articlesConnection!
+
+  const articles = extractNodes(articlesConnection)
+
+  const [page, setPage] = useState(1)
+  const [isLoading, setIsLoading] = useState(false)
+  const logger = createLogger("FairOrganizerDedicatedArticles.tsx")
+
+  useEffect(() => {
+    setIsLoading(true)
+
+    const relayRefetchParams = {
+      id: slug,
+      first: PAGE_SIZE,
+      page,
+    }
+
+    relay.refetch(relayRefetchParams, null, error => {
+      if (error) {
+        logger.error(error)
+      }
+
+      setIsLoading(false)
+    })
+  }, [page])
+
+  const loadNext = () => {
+    const currentPage = page ?? 0
+    const nextPage = currentPage + 1
+    if (hasNextPage) {
+      loadPage(nextPage)
+    }
+  }
+
+  const loadPage = newPage => {
+    setPage(newPage)
+    updateUrl({ page: newPage })
+  }
+
+  return (
+    <>
+      <Title>{`${name} | Artsy`}</Title>
+
+      <DedicatedArticlesBreadcrumbs fairOrganizer={fairOrganizer} />
+
+      <Box id="jump--FairOrganizerDedicatedArticles">
+        <Text as="h1" variant="lg" mt={6}>
+          All Articles for Explore {name} on Artsy
```

Comment:
**suggestion**: I clarified with Jonathan in [this Slack thread](https://artsy.slack.com/archives/C9SATFLUU/p1628761854365900?thread_ts=1628761584.365100&cid=C9SATFLUU) and we should remove "Explore" from this header text.

Reply:
Oh that makes sense 😅  Thanks!